### DISCUSSION TITLE
2009/05/22/%E9%BC%A0%E6%A0%87%E9%85%8D%E5%90%88ctrl%E9%94%AE%E7%9A%84%E5%A6%99%E7%94%A8

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#265] [Announcements]
# 2009/05/22/%E9%BC%A0%E6%A0%87%E9%85%8D%E5%90%88ctrl%E9%94%AE%E7%9A%84%E5%A6%99%E7%94%A8

LabVIEW中有众多的快捷键，这些快捷键可以大大加快编程速度。不过记住这些快捷键本身也是个挑战，比如说我吧，我只能记住几个最常用的快捷键，比如 Ctrl+C，Ctrl+V 等。我编程序主要还是依赖鼠标，单一个鼠标应付各种工作还是显得有些吃力的，编程速度可能没有熟练使用快捷键那么迅速。编的程序多了，不经意间就发现了一些鼠标配合Ctrl键的妙用，于是就把最有用的几种用法记录下来了。

https://blog.qizhen.xyz/2009/05/22/%E9%BC%A0%E6%A0%87%E9%85%8D%E5%90%88ctrl%E9%94%AE%E7%9A%84%E5%A6%99%E7%94%A8

**Comment by ruanqizhen** [comment]
之前博客下的留言：

天道
2009年05月29日 09:14 编辑
学习了 复制那些熟悉点 调换函数输入参数的连线还真不知道

回复
NeoYu
2009年05月27日 13:28 编辑
多谢分享，不过，好像在群里的Ian和Brain说过了，我当时问过“如何复制属性节点？”

回复
